projects/adapter/src/dbt/adapters/fal_experimental/utils/environments.py

```diff
@@ -311,15 +310,17 @@ def _version_is_prerelease(raw_version: str) -> bool:
     return package_version.is_prerelease
 
 
-def _get_adapter_root_path() -> Optional[Path]:
-    import dbt.adapters.fal as adapter
+def _get_project_root_path(pacakge: str) -> Path:
```

Comment:
small typo `s/pacakge/package`

```suggestion
def _get_project_root_path(package: str) -> Path:
```